mozilla_django_oidc/views.py

```diff
@@ -114,15 +114,15 @@ def redirect_url(self):
 
     def dispatch(self, request, *args, **kwargs):
         """Log out the user."""
-
-        if request.user.is_authenticated():
+        # Check if a method exists to build the URL to log out the user
+        # from the OP.
+        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
```

Comment:
Oh you're right! It shouldn't. That means that only `logout_url = self.redirect_url` can be used if the user is already anonymous. I'll update.